.github/openapi/python-config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "generatorName": "python",
+  "packageName": "Scicat-Python-SDK",
```

Comment:
If this maps to the generated python package name,
I would prefer a name in all lower case and without -/_, eg. `scicatsdk`

Reply:
That's a good point, and the naming should be consistent for all the SDK of different languages.

Reply:
Apologies for the delayed response. It would be preferable to use - in the naming, such as “scicat-sdk-python” or “scicat-sdk-ts,” as it improves readability by clearly separating words, aligns with common naming conventions in both Python and TypeScript, and makes the package’s purpose easily identifiable for developers.